vars/deleteFile.groovy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/groovy
+
+@NonCPS
+def call(String path) {
```

Comment:
can't help here, since I don't know any alternative... if it works: let's do it :) we can always improve if we gain more knowledge and find a better way